timetracker/timetracker/urls.py

```diff
@@ -16,10 +16,15 @@
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.http import HttpResponse
+from django.shortcuts import render
+from .forms import NameForm
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+    # How is the request module understood if it's not imported above?
+    form = NameForm(request.GET)
```

Comment:
Here `request` isn't a module, but a variable that was passed into the `hello_world` function.

It's an instance Django's HttpRequest class that gets passed to all function based views automatically.